### ISSUE TITLE
price not displayed in embed

### ISSUE THREAD

**Issue by typoooo** [devjannis/vinted-monitor#1]
The price is displayed as 0.0, the log says the price is unable to be obtained

**Comment by 0xJannis** [comment]
Is this problem occurring with every product?

**Comment by typoooo** [comment]
yes, ive tried to fix it myself but i cant figure out what it is.
